css/sitemapStyles.css

```diff
@@ -1,90 +1,119 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,700;0,900;1,900&display=swap");
+
 *{ 
     box-sizing:border-box; 
+    margin: 0;
+    padding : 0;
+    font-family: Poppins;
 }
 body{ 
-    background:#001d1a; 
-    max-width:980px; 
-    margin:0 auto; 
-    padding:5px; 
-    font: 14px "Arial", Tahoma, sans-serif; 
+    background:#000000; 
```

Comment:
Don't use black background it is never recommended. Choose the existing color scheme.